doc/cabal-project.rst

```diff
@@ -194,8 +194,8 @@ Specifying Packages from Remote Version Control Locations
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Starting with Cabal 2.4, there is now a stanza
-``source-repository-package`` for specifying packages from an external
-version control.
+``source-repository-package`` for specifying remote packages that cabal will vendor from an external version control system.
```

Comment:
Is it possible to avoid the word "vendor"? I have met several people who were confused by it.

Reply:
would 'use' , or 'consider during remote package retrieval' work better?

Reply:
Use sounds fine. The longer option would have “during solving” rather than “during retrieval”, I think, but I’m not sure it’s necessary to specify the point in time…